cron/tz.go

```diff
@@ -0,0 +1,26 @@
+package cron
+
+import (
+	"time"
+)
+
+// Schedule wrapper with time zone awarness
+type timezoneAwareSchedule struct {
+	underlingSchedule Schedule
```

Comment:
underlying?

Reply:
totally, sorry I will fix that later on this week.  I am quite buzzy right now.